rpc/config.go

```diff
@@ -23,13 +24,20 @@ type ProfilerConfig struct {
 	ListenAddress string
 }
 
+type GRPCConfig struct {
```

Comment:
As comment above I think we can probably collapse these into a `ServerConfig` in `rpc/config.go`